api/SVGAnimateColorElement.json

```diff
@@ -44,7 +44,7 @@
             "version_removed": "2.0"
           },
           "webview_android": {
-            "version_added": "1",
+            "version_added": "≤37",
```

Comment:
This and other cases with `"version_removed": "37"` will end up looking broken on MDN due to https://github.com/mdn/yari/issues/3238. It's not so broken that I think it has to change, but if you can find the right version for this now, that would be better.